server/handlers/static/bucket.js

```diff
@@ -94,21 +66,18 @@ async function bucketHandler(req, res, next) {
     filePath = path.join(filePath, 'index.html');
   }
 
-  // TODO: For production, lookup if the file exists in
-  // the commit manifest, before trying to stream it
-
   try {
     const file = await storage
       .bucket(bucketName)
-      .file(path.join(activeCommit, filePath));
+      .file(path.join(folder, filePath));
 
     const [metadata] = await file.getMetadata();
 
     res.setHeader('content-type', metadata.contentType);
     file.createReadStream().pipe(res);
   } catch (e) {
     console.error('Could not serve', filePath);
-    return next();
+    res.send(`Could not send ${filePath} due to ${e}.`);
```

Comment:
This is probably fine for dev / stage, but if we're about to use this handler in prod at some point we should just return 404/50X

Reply:
Yep! Will create a bunch of follow-up issues once this is in, like response times can also be improved in general, redirect handlers need to work with Cloud Storage, ...